tests/jobsapp/test_endpoints.py

```diff
@@ -0,0 +1,15 @@
+import pytest
+from rest_framework.test import APIClient
+
+
+@pytest.fixture
+def api_client():
+    return APIClient()
+
+
+def test_contact_us(api_client):
+    resp = api_client.get("/api/contact-us")
+    assert resp.status_code == 405
+    data = {"name": "test", "email": "test@tester.cat", "subject": "Testing", "message": "tests"}
+    resp = api_client.post("/api/contact-us", data=data)
+    assert resp.status_code == 202
```

Comment:
You can also check if email has been sent.

```python
from django.core import mail
assert len(mail.outbox) == 1
```
But you need celery works syncron.

```
    CELERY_TASK_ALWAYS_EAGER = True
```

You can override django's settings in `pytest_configure`  in `conftest` file.